tests/test_ldbc.py

```
@@ -0,0 +1,426 @@
+    },
+]
+
+
+def test_load_csv():
+    total_time = 0
+    for file in node_files:
+        common.g.query(f"CREATE INDEX FOR (n:{file['label']}) ON n.id")
+        query = f"""
+            LOAD CSV WITH HEADERS DELIMITER '|' FROM $file AS row
+            RETURN count(row)
+            """
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)